package.json

```diff
@@ -26,6 +26,7 @@
     "rimraf": "^2.6.2"
   },
   "dependencies": {
+    "vscode-languageserver-protocol-foldingprovider": "^1.0.1",
```

Comment:
the language service should not depend on the protocol modules as that one depends on json-rcp with 'fs' and other node dependencies.
We just need the types (FoldingRangeList, FoldingRange), which we can define ourselves in the css service
(at some point these types will go to the languageserver-types module)